plinth/modules/dynamicdns/templates/dynamicdns_configure.html

```diff
@@ -0,0 +1,168 @@
+{% extends "base.html" %}
+{% comment %}
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+{% endcomment %}
+
+{% load bootstrap %}
+
+{% block content %}
+
+  <form class="form" method="post">
+    {% csrf_token %}
+    <div id='dynamicdns-no-js'>
```

Comment:
I'd use the official way here, the `<noscript>hej you have JS disabled!</noscript>` tag. Everything inside will only be displayed when javascript is disabled and you don't need to use additional javascript to hide it.